examples/handlers/aop.links

```diff
@@ -6,7 +6,7 @@ sig runState :
         (Comp ({Get:(()) {}-> s,Put:(s) {}-> ()|e}, a)) ->
   (s) -> Comp ({Get{_},         Put{_}         |e}, (a, s))
 fun runState(m)(s)() {
-  (open handle(m) {
```

Comment:
it looks like you committed some unrelated other changes here

Reply:
Yes, I fixed a broken example too.